toponetx/datasets/graph.py

```diff
@@ -0,0 +1,78 @@
+"""Various examples of named graphs represented as complexes."""
+
+import networkx as nx
+import numpy as np
+
+from toponetx import CellComplex
+from toponetx.algorithms.spectrum import (
+    hodge_laplacian_eigenvectors,
+    set_hodge_laplacian_eigenvector_attrs,
+)
+from toponetx.transform.graph_to_simplicial_complex import graph_2_clique_complex
+
+__all__ = ["karate_club_complex"]
+
+
+def karate_club_complex(complex_type="simplicial"):
```

Comment:
Rename to `karate_club` only, as the other data functions do not use the word _complex.